templates/test-sw.hbs

```diff
@@ -44,22 +44,22 @@
 
     <script>
       mocha.setup({
-        ui: 'bdd'
+        ui: 'bdd',
+        timeout: 10000,
```

Comment:
Why set a globally long timeout when individual tests can (and IMO should) define the timeout. i.e. if  a test should pass / fail quickly, this will unnecessarily extend the test time.